input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+        </tr>
+        <tr>
+            <td style="width: 33.33%;"><a href="StructureDefinition-au-core-medication.html">AU Core Medication</a></td>
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-location.html">US Core Location</a></td>
+            <td style="width: 33.33%;">Medication.code binding differs: US Core extensibly binds to RxNorm (which is not an AU medicines terminology) vs AU Core preferred Australian Medication and PBS Item Codes.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html">AU Core Organization</a></td>
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-organization.html">US Core Organization</a></td>
+            <td style="width: 33.33%;">Organization.active minimum cardinalities differ: US Core requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-patient.html">US Core Patient</a></td>
+            <td style="width: 33.33%;">Gender identity extension differs: US Core defines an US Core Gender Identity extension vs AU Core which requires the use of the International Gender Identity extension.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html">AU Core Practitioner</a></td>
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitioner.html">US Core Practitioner</a></td>
+            <td style="width: 33.33%;">Practitioner.identifier minimum cardinalities differ: US Core requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html">AU Core PractitionerRole</a></td>
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitionerrole.html">US Core PractitionerRole</a></td>
+            <td style="width: 33.33%;">Non-compliance due to constraint pd-1: US Core requires constraint pd-1, which mandates that resources SHALL have contact information or a reference to an Endpoint, whereas AU Core does not enforce this requirement.</td>
+        </tr>
+    </tbody>
+</table>
+
+##### Additional profiles 
+
+AU Core has no equivalent for the following profiles defined in the US Core:
```

Comment:
Suggest reword this sentence to be "This version of AU Core has no equivalent profile for the following US Core profiles:"

Reply:
Done